src/content/docs/self-hosting/kubernetes.mdx

```diff
@@ -26,7 +26,7 @@ stringData:
   ATUIN_HOST: "127.0.0.1"
   ATUIN_PORT: "8888"
   ATUIN_OPEN_REGISTRATION: "true"
-  ATUIN_DB_URI: "postgres://atuin:seriously-insecure@localhost/atuin"
+  ATUIN_DB_URI: "postgres://atuin:seriously-insecure@postgres/atuin"
```

Comment:
Postgres is now accessible via the `postgres` service and no longer in the same pod.